### PR TITLE
Migrate shell modals to PF4

### PR DESCRIPTION
**Pull request by marusak**
There are still two remaining ones. Both use `pkg/lib/machine-dialogs.js` which cannot be simple converted into PF4.
@mvollmer mentioned some time ago that we shoud move all of these into dashboard (which also uses them) and implement the ones we need in host switcher in react. I think that is the approach in which we want to go.

Tests are going to be very sad, just want to get a list of failing tests that I can fix locally first.

### CODE REVIEW

File: test/verify/check-apps [hunk 1]
```diff
@@ -177,7 +182,7 @@ class TestApps(PackageCase):
         b.wait_present(".app-description:contains('DESCRIPTION:de')")
         b.wait_not_present(".app-description:contains('DESCRIPTION:none')")
 
-        set_lang("ja-ja")
```
Comment (KKoukiou) [inline test/verify/check-apps:180]: Hmm how is this change related?

File: pkg/shell/index.html [hunk 2]
```diff
@@ -126,234 +103,193 @@ <h4 class="modal-title" translate="yes">About Web Console</h4>
...
-                                                        <button class="credential-change pf-c-button pf-m-primary"
-                                                            translate="yes">Change password</button>
-                                                    </dd>
-                                                </dl>
-                                            </div>
-                                        </td>
-                                    </tr>
-                                </tbody>
-                            </table>
+        <div hidden aria-hidden="true" id="credentials-dialog">
```
Comment (KKoukiou) [inline pkg/shell/index.html:112]: This modal has the same issue as described here https://github.com/cockpit-project/cockpit/issues/14895

In this case tabbing works immediately, but is escapes the modals after the action buttons.
I think we can fix it in another PR, since this issue is now in many places.

File: test/verify/check-apps (hunk 1 above)
Comment (marusak) [reply test/verify/check-apps:180]: `ja-ja` is not a valid locale. It somehow worked before, but is broken now. `ja-jp` is the expected locale. Originally fixed here: https://github.com/cockpit-project/cockpit/commit/a395a4a5155e62b24cdb8afa3697618c31282e1f

I could split it to separate commit if you would prefer that.

File: pkg/shell/index.html (hunk 2 above)
Comment (marusak) [reply pkg/shell/index.html:112]: Yeah, I think lets do it in separate PR where we fix it in all places at once.

File: test/verify/check-apps (hunk 1 above)
Comment (KKoukiou) [reply test/verify/check-apps:180]: It's fine - I just thought it was a typo.

**Review by KKoukiou — APPROVED** [review]
